extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -585,8 +587,10 @@ private JvmBuildTargetEx getJvmTarget(List<BuildTarget> buildTargets) throws Cor
         }
 
         if (StringUtils.isBlank(jvmTarget.getJavaHome()) || StringUtils.isBlank(jvmTarget.getGradleVersion())) {
-            throw new CoreException(new Status(IStatus.ERROR, ImporterPlugin.PLUGIN_ID,
-                    "Invalid JVM build target: " + jvmTarget.toString()));
+            JavaLanguageServerPlugin.logException(
+                new CoreException(new Status(IStatus.WARNING, ImporterPlugin.PLUGIN_ID,
+                    "Invalid JVM build target: Missing JavaHome or GradleVersion."))
```

Comment:
`Empty Java home or Gradle version in JVM target`

Reply:
Addressed in https://github.com/microsoft/vscode-gradle/pull/1594/commits/974ebc4e114459b65b61c7e8c95634986511ae9f